src/js/components/Calendar/StyledCalendar.js

```diff
@@ -61,35 +73,50 @@ const slideStyle = props => {
   `;
 };
 
+const weeksSizeStyle = props => {
+  return css`
+    ${props.fillContainer && 'display: flex;'}
+    ${props.fillContainer && 'flex-direction: column;'}
+    ${props.fillContainer && 'height: 100%;'}
+  `;
+};
 const StyledWeeks = styled.div`
   position: relative;
+  ${props => weeksSizeStyle(props)}
```

Comment:
Would be a bit simpler to:
`${props => props.fillContainer && weeksSizeStyle()`

Reply:
Sounds good. Update made.